vssexporters/vss2json.py

```diff
@@ -12,17 +12,22 @@
 import json
 from vspec.model.vsstree import VSSNode, VSSType
 
-def add_arguments(parser: argparse.ArgumentParser):
-    parser.add_argument('--json-pretty', action='store_true', help=" Pretty print JSON output.")
 
+def add_arguments(parser: argparse.ArgumentParser):
+    parser.add_argument('--json-no-extended-attributes',
+                        action='store_true', help="Generate no extended attributes.")
+    parser.add_argument('--json-all-extended-attributes', action='store_true',
+                        help="Generate all extended attributes found in the model (default is generating only those given by the -e/--extended-attributes parameter).")
```

Comment:
1. I think we do need them for each exporter because not all arguments do make sense for all exporters (i.e. generate "all" is quite useful for json and yaml, but not useful with CSV maybe, at least not easy, because it requires a two-pass iteration over the data model.
2. Looking at it, I agree the "no-extended" seems redundant: With "-e"=Please use this list, with "-all": Generate everyting you can. With -e unset it is basically none